internal/document/editor/editorservice/service.go

```diff
@@ -58,6 +58,17 @@ func (s *parserServiceServer) Deserialize(_ context.Context, req *parserv1.Deser
 			Cwd:         notebook.Frontmatter.Cwd,
 			SkipPrompts: notebook.Frontmatter.SkipPrompts,
 		}
+
+		var runme *parserv1.Runme
+
+		if notebook.Frontmatter.Runme.ID != "" && notebook.Frontmatter.Runme.Version != "" {
```

Comment:
@pastuxso I think we want to set ID and Version separately since, from a validity standpoint, they can show up either/or or both.

Otherwise :+1:.